shoop/core/management/commands/shoop_extractmessages.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Extract translatable strings from shoop
+"""
+from __future__ import unicode_literals
+
+import logging
+
+import os
+from _misc import sanity_utils
+from _misc.gather_i18n import MAPPING, extract_from_file, write_po_wrap
```

Comment:
shoop should not depend on _misc.

The tooling used by this management command should be moved under shoop.utils or something like that.